.github/workflows/gradle.yml

```diff
@@ -49,34 +53,36 @@ jobs:
       run: bash <(curl -s https://codecov.io/bash) -f ./build/reports/jacoco/test/jacocoTestReport.xml
     - name: Publish RCA jar to maven local
       working-directory: ./tmp/performance-analyzer-rca
-      run: ./gradlew publishToMavenLocal
-    - name: Build PA gradle using the new RCA jar
-      working-directory: ./tmp/pa
-      run: rm -f licenses/performanceanalyzer-rca-1.0.0.0.jar.sha1
-    # PA depends on OpenSearch. Checkout and publish to maven local for PA build.
-    - name: Checkout OpenSearch
+      run: ./gradlew publishToMavenLocal -Dopensearch.version=1.1.0-SNAPSHOT
+
+    # PA in ./tmp/performance-analyzer
+    - name: Checkout Performance Analyzer
       uses: actions/checkout@v2
       with:
-        repository: 'opensearch-project/OpenSearch'
-        path: OpenSearch
-        ref: '1.0'
-    - name: Build OpenSearch
-      working-directory: ./OpenSearch
-      run: ./gradlew publishToMavenLocal -Dbuild.snapshot=false
+        repository: dblock/performance-analyzer
+        ref: fix-snapshot-build
```

Comment:
We might need to change this before the merge

Reply:
There's a circular dependency between these two projects, so we cannot do that without breaking everything. I will update to use main branches after both are in.